book/src/users/startup.md

```diff
@@ -0,0 +1,65 @@
+# Startup
+
+Configuration occurs at startup via flags:
+
+```
+cargo run -p trin -- --flag1 flag1value --flag2 flag2value
+```
+Backslashes allow flags to be on new lines for easier reading.
+```
+cargo run -p trin -- \
+    --flag1 flag1value \
+    --flag2 flag2value \
+    --flag3 flag3value1,flag3value2 \
+```
+
+## Flags
+
+For the most up to date flags run:
+
+```
+cargo run -p trin -- --help
+```
+
```

Comment:
I think it's worthwhile to mention here that to quickly connect to the testnet, users can use `--bootnodes default` flag